webapp/loggedin/modules/home/dashboard/activityLog/activityLogMessageParser/isItemDeletedFns/isItemDeletedFnsRecord.js

```diff
@@ -0,0 +1,9 @@
+import * as ActivityLog from '@common/activityLog/activityLog'
+
+export default {
+  [ActivityLog.type.recordCreate]: () => activityLog =>
+    ActivityLog.isRecordDeleted(activityLog),
+
+  [ActivityLog.type.recordStepUpdate]: () => activityLog =>
+    ActivityLog.isRecordDeleted(activityLog),
+}
```

Comment:
this can simply be:
```
_isRecordDeleted = R.pipe(ActivityLog.getRecordUuid, R.isNil)

export default {
    [ActivityLog.type.recordCreate]: () => _isRecordDeleted,
 
    [ActivityLog.type.recordStepUpdate]: () => _isRecordDeleted,
  }
```

Reply:
perfect, many thanks